.github/workflows/linter.yml

```diff
@@ -1,102 +1,102 @@
-name: Linter
```

Comment:
Feel free to delete this file, we can always have access to it via the git history